impeller/entity/entity_pass.cc

```diff
@@ -896,8 +923,17 @@ void EntityPass::SetClearColor(Color clear_color) {
   clear_color_ = clear_color;
 }
 
-Color EntityPass::GetClearColor() const {
-  return clear_color_;
+Color EntityPass::GetClearColor(ISize target_size) const {
+  Color result = clear_color_;
```

Comment:
Any reason we should keep the `clear_color_`? Can we just remove it along with `SetClearColor` now that we're handling everything below Aiks?

Reply:
I'll look into it.  I considered it, but since it didn't show up as a requirement to get the tests passing didn't pursue it.

Reply:
Done